prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -431,6 +431,7 @@ has_detector FDD && ! has_detector_from_global_reader FDD && has_processing_step
 has_detector FV0 && ! has_detector_from_global_reader FV0 && has_processing_step FV0_RECO && add_W o2-fv0-reco-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC"
 has_detector ZDC && ! has_detector_from_global_reader ZDC && has_processing_step ZDC_RECO && add_W o2-zdc-digits-reco "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC"
 has_detector HMP && ! has_detector_from_global_reader HMP && has_processing_step HMP_RECO && add_W o2-hmpid-digits-to-clusters-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT --pipeline $(get_N HMP-Clusterization HMP REST 1 HMPCLUS)"
+has_detector HMP && ! has_detector_from_global_reader HMP && has_processing_step HMP_RECO && add_W o2-hmpid-matcher-workflow "--disable-root-input $DISABLE_ROOT_OUTPUT"
```

Comment:
@gvolpe79, in the matcher you are requesting ITS-TPC matching MC info, which is not available during raw data processing (which FST does). This line should look like
```
has_detector HMP && ! has_detector_from_global_reader HMP && has_processing_step HMP_RECO && add_W o2-hmpid-matcher-workflow "--disable-root-input $DISABLE_ROOT_OUTPUT $DISABLE_MC"
```

Reply:
@shahor02 , thanks! I have just modified dpl-workflow.sh!

Reply:
hi @shahor02 now the problem is again on the aod-producer-workflow....!!